src/platform/debugger/multiplexingDebugService.ts

```diff
@@ -78,21 +84,41 @@ export class MultiplexingDebugService implements IJupyterDebugService {
         return this.activeService.onDidChangeBreakpoints;
     }
     public get onBreakpointHit(): Event<void> {
+        if (!this.jupyterDebugService) {
+            throw new Error('No jupyter debugger service');
+        }
         return this.jupyterDebugService.onBreakpointHit;
     }
     public startRunByLine(config: DebugConfiguration): Thenable<boolean> {
         this.lastStartedService = this.jupyterDebugService;
+        if (!this.jupyterDebugService) {
+            throw new Error('No run by line debugger service');
+        }
         return this.jupyterDebugService.startRunByLine(config);
     }
     public registerDebugConfigurationProvider(debugType: string, provider: DebugConfigurationProvider): Disposable {
         const d1 = this.vscodeDebugService.registerDebugConfigurationProvider(debugType, provider);
-        const d2 = this.jupyterDebugService.registerDebugConfigurationProvider(debugType, provider);
-        return this.combineDisposables(d1, d2);
+        if (this.jupyterDebugService) {
+            const d2 = this.jupyterDebugService.registerDebugConfigurationProvider(debugType, provider);
+            return this.combineDisposables(d1, d2);
+        }
+        return {
```

Comment:
I believe you can just return d1 in this case, not rewrite it to be an anonymous object that calls d1.dispose.